RecessApplication/settings.py

```diff
@@ -24,13 +24,19 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = '_4=x29q$v0$kg862o5k7-)i-ty06e#$=45k$pm@m^cui*z@%n7'
 
+# SECURITY WARNING: keep the secret key used in production secret!
```

Comment:
Do we have a better way to do this right now?

Reply:
I can always regenerate these so for now it might not be a huge matter

Reply:
We should probably have it read local environment variables that are only stored on our computer and not in source control. Or just in a separate file that we add to the .gitignore